userspace/libscap/test/scap_event.ut.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include "scap.h"
+#include <gtest/gtest.h>
+
+TEST(scap_event, empty_clone)
+{
+    char scap_error[SCAP_LASTERR_SIZE];
+    struct scap_sized_buffer decoded_params[PPM_MAX_EVENT_PARAMS];
+    uint32_t status, n, expected_n;
+    scap_evt *evt;
+
+    status = scap_event_generate(&evt, scap_error, PPME_SYSCALL_CLONE_20_E, 0);
+    expected_n = 0;
+    ASSERT_EQ(status, SCAP_SUCCESS) << "scap_event_generate failed with error " << scap_error;
+    EXPECT_EQ(evt->nparams, expected_n);
+
+    n = scap_event_decode_params(evt, decoded_params);
+    EXPECT_EQ(n, expected_n);
```

Comment:
I'm not 100% familiar with this function, but don't we want to check any value from the `decoded_params`?

Reply:
Good catch 👍

Reply:
added more expects